api/src/main/java/org/hawkular/inventory/api/MultipleEntityBrowser.java

```diff
@@ -23,7 +23,7 @@
  * @author Lukas Krejci
  * @since 1.0
  */
-interface MultipleEntityBrowser<Entity> {
+public interface MultipleEntityBrowser<Entity> {
```

Comment:
It allows for more general form of use. Types are good, but something more abstract what returns the entitites might be useful also for the API itself. I agree, the word `Browser` might expose some internal terminology, what about something more general name like `HasEntities` with a single method `entities()` ?